sample.wdl

```diff
@@ -125,6 +126,19 @@ workflow Sample {
         metrics.picardMetricsFiles, metrics.rnaMetrics,
         metrics.targetedPcrMetrics, [markdup.metricsFile], flatten(qc.reports)]
 
+    call structuralVariantCalling.SVcalling as svCalling{
```

Comment:
I feel like this should be optional. I suspect this can take quite some time, so if a researcher isn't interested in this they should be able to disable it. 

I also feel like this should probably be disabled in the somatic pipeline, as this is a germline analysis.

Reply:
Should I move it to germline.wdl otherwise?

Reply:
Yeah, you could add it to the same scatter as sample.wdl gets called in.